docker/linea-mainnet/besu/linea-besu.config.toml

```diff
@@ -4,7 +4,7 @@ genesis-file="/var/lib/besu/genesis.json"
 data-path="/opt/besu/data"
 
 # Sync mode and data layer implementation
-sync-mode="FULL"
+sync-mode="SNAP"
```

Comment:
Any particular reason to use FULL sync mode for mainnet besu? if not, I will change it to SNAP here